tier1-icons.html

```diff
@@ -302,10 +302,10 @@
         <g id="reading">
```

Comment:
I wasn't sure if an RTL book would have the bookmark on the opposite side, so I left it.